kernel/fatal.c

```diff
@@ -94,9 +94,6 @@ void z_fatal_error(unsigned int reason, const z_arch_esf_t *esf)
 	struct k_thread *thread = IS_ENABLED(CONFIG_MULTITHREADING) ?
 			_current : NULL;
 
-	/* twister looks for the "ZEPHYR FATAL ERROR" string, don't
```

Comment:
actually Twister looks for `ZEPHYR FATAL ERROR`,
not sure it is really good way to keep it consistent relaying on this comment here though

https://github.com/zephyrproject-rtos/zephyr/blob/a0b746ed74f373192dc524333b8a496f7d3dae99/scripts/pylib/twister/twisterlib/harness.py#L32

Reply:
Oh, I missed this. Will keep the comment then, thanks :)